.github/dependabot.yml

```diff
@@ -7,31 +7,10 @@ updates:
     time: "03:00"
     timezone: Europe/Oslo
   open-pull-requests-limit: 99
-  ignore:
-  - dependency-name: "@types/node"
-    versions:
-    - ">= 13.a, < 14"
-  - dependency-name: "@types/node"
-    versions:
-    - ">= 14.a, < 15"
```

Comment:
We probably want `@types/node` and `typescript` to stay. They should be explicitly updated by a developer. See this example: https://github.com/pulumi/actions/blob/master/.github/dependabot.yml